lib/partition_setup.pm

```diff
@@ -63,6 +78,10 @@ sub wipe_existing_partitions_storage_ng {
 }
 
 
+=head2 create_new_partition_table
+
+=cut
+
```

Comment:
Please remove the empty line between `=cut` and the `sub …`

Reply:
According to the coding style in the parent ticket we want this line?

Reply:
We did it without that empty line everywhere so I updated the parent ticket.

Reply:
Adaption will come soonish